### ISSUE TITLE
Error when marking channel as read

### ISSUE THREAD

**Issue by stoatally** [crmarsh/discord-linux-bugs#33]
```
Uncaught Exception:
TypeError: Error processing argument at index 0, conversion failure from /opt/discord-canary/resources/app.asar/app/images/linux-tray-unread.png
    at TypeError (native)
    at SystemTray.setBadge (/opt/discord-canary/resources/app.asar/app/SystemTray.js:117:21)
    at EventEmitter.<anonymous> (/opt/discord-canary/resources/app.asar/app/SystemTray.js:95:13)
    at emitOne (events.js:90:13)
    at EventEmitter.emit (events.js:182:7)
    at EventEmitter.<anonymous> (/opt/discord-canary/resources/electron.asar/browser/api/web-contents.js:128:25)
    at emitTwo (events.js:100:13)
    at EventEmitter.emit (events.js:185:7)
```

Running 0.0.7 on Arch Linux.

**Comment by crmarsh** [comment]
So strange that this works for some people and not others :-/

**Comment by crmarsh** [comment]
I think this is fixed as of 0.0.11
